src/test/java/com/fewlaps/quitnowemailsuggester/EmailSuggestorTest.java

```diff
@@ -341,4 +341,19 @@ public void shouldFixGmaikDotCom() throws InvalidEmailException {
     public void shouldFixYahou() throws InvalidEmailException {
         assertEquals("roc@yahoo.com", es.getSuggestedEmail("roc@yahou.com"));
     }
+
+    @Test
+    public void shouldNotFix() throws InvalidEmailException {
```

Comment:
*shouldNotFix* is something that the library has to do? Or you're trying to say "shouldReturnTheSame_whenTheEmailIsValid"?